SH4RE/SH4RE/Account/AccountView.swift

```diff
@@ -175,6 +175,10 @@ struct AccountView: View {
                                 }
                             }
                         }
+                        
+                        getUserRating(uid: getCurrentUserUid(), completion: { rating in
+                            numberOfStars = rating
+                        })
```

Comment:
![Screenshot 2023-03-29 at 7 01 03 PM](https://user-images.githubusercontent.com/46950469/228686829-eb07fd94-744a-4609-8482-f4110c216a3d.png)
i doubt you'll come across it again but just as an FYI, this is equivalent to this I believe.